### PR TITLE
Feature/run eslint

### PR DESCRIPTION
**Pull request by ashmit-coder**
Fixes: #89 
Ran the linter and also added a worklfow for the same
refrence: #94

### CODE REVIEW

**Comment by ashmit-coder** [comment]
Hey @PradyumnaKrishna i added the workflow as you demanded

File: .github/workflows/run_eslint.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+          node-version: 18
+
+      # ESLint and Prettier must be in `package.json`
+      - name: Install Node.js dependencies
+        working-directory: ./client
+        run: npm ci
+
+      - name: Run linters
+        working-directory: ./client
+        run : npm run lint-fix
```
Comment (PradyumnaKrishna) [inline .github/workflows/run_eslint.yml:32]: Minor nit and then we are ready to go

```suggestion
        run : npm run lint
```

**Comment by ashmit-coder** [comment]
@PradyumnaKrishna done with the changes

**Review by ashmit-coder — COMMENTED** [review]
completed changes as mentioned

**Review by PradyumnaKrishna — APPROVED** [review]

**Pull request merged**